### DISCUSSION TITLE
如paopaodns配合paopaogateway，paopao-pref生成的缓存是否没用

### DISCUSSION THREAD

**Discussion by jaesony** [kkkgo/PaoPao-Pref#5] [提问和帮助] [answered]
国外应该都是fakeip，由节点解析。paopao-pref生成的真实IP缓存是否有影响？

**Comment by kkkgo** [comment, marked as answer]
只有真实IP结果才会进入redis缓存。
在`AUTO_FORWARD_CHECK`为`yes`的情况下，会先校验真实IP再转发fakeip。校验过程会使用缓存。

**Reply by jaesony** [reply]
懂了，是要确定IP属于国外再转发，谢谢！！！
